Apromore-Custom-Plugins/User-Admin-Portal-Plugin/src/main/java/org/apromore/plugin/portal/useradmin/UserAdminController.java

```diff
@@ -808,6 +809,7 @@ private Group setSelectedGroup(Group group) {
         nonAssignedUserModel.setMultiple(true);
         nonAssignedUserList = new AssignedUserListbox(nonAssignedUserListbox, nonAssignedUserModel, "Users not in the group");
         selectedGroup = group;
+        isGroupDetailDirty = false; // extra defensive, ensure dirty is not set by field's setValue
```

Comment:
Similarly, would it make sense to remove the duplicate `isGroupDetailDirty = false;` on line 784?